x/account/model/account.go

```diff
@@ -13,16 +13,18 @@ import (
 type AccountInfo struct {
 	Username       types.AccountKey `json:"username"`
 	CreatedAt      int64            `json:"created_at"`
-	ResetKey       crypto.PubKey    `json:"reset_key"`
+	SignningKey    crypto.PubKey    `json:"signning_key"`
```

Comment:
To be able to parse IR from last version, need to add following to `ir.go`. 
```
// AccountInfoV1 - user information
type AccountInfoV1 struct {
	Username       types.AccountKey `json:"username"`
	CreatedAt      int64            `json:"created_at"`
	ResetKey       crypto.PubKey    `json:"reset_key"`
	TransactionKey crypto.PubKey    `json:"transaction_key"`
	AppKey         crypto.PubKey    `json:"app_key"`
}

// AccountBankV1 - user balance
type AccountBankV1 struct {
	Saving          types.Coin    `json:"saving"`
	CoinDay         types.Coin    `json:"coin_day"`
	FrozenMoneyList []FrozenMoney `json:"frozen_money_list"`
}

// AccountMetaV1 - stores tiny and frequently updated fields.
type AccountMetaV1 struct {
	Sequence             uint64     `json:"sequence"`
	LastActivityAt       int64      `json:"last_activity_at"`
	TransactionCapacity  types.Coin `json:"transaction_capacity"`
	JSONMeta             string     `json:"json_meta"`
	LastReportOrUpvoteAt int64      `json:"last_report_or_upvote_at"`
	LastPostAt           int64      `json:"last_post_at"`
}

```
and change AccountRowIR to 
```
type AccountRowIR struct {
	Username            types.AccountKey      `json:"username"`
	Info                AccountInfoV1           `json:"info"`
	Bank                AccountBankV1           `json:"bank"`
	Meta                AccountMetaV1           `json:"meta"`
	Reward              Reward                `json:"reward"`
	PendingCoinDayQueue PendingCoinDayQueueIR `json:"pending_coin_day_queue"`
}

```

Reply:
added V1 to prevent conflict